test-suite/test_utils.h

```diff
@@ -0,0 +1,81 @@
+#ifndef MPIBIND_TEST_UTILS_H
+#define MPIBIND_TEST_UTILS_H
+#include <stdbool.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "mpibind.h"
+#include "mpibind-priv.h"
```

Comment:
mpibind-priv.h should not be used by user programs. The only header for users/tests is 'mpibind.h'

```suggestion
#include "mpibind-priv.h"
```

Reply:
This has been removed from test_utils.h